src/views/organisationType.njk

```diff
@@ -1,5 +1,58 @@
 {% extends 'base.njk' %}
```

Comment:
I think we can format this file, i just pulled the file down on my local and right clicked on this file > format document... that should tidy up some of the linting

Reply:
I tried but i don't think anything happens when I do

Reply:
Ah ok, so when i click format there's a few tab spaces entered... but its nothing to block this ticket from being merged